# List of different user profiles used for simulating HRI

profile:

################################################
############   ENGAGED PROFILES   ##############
################################################

  - id: engaged_complete # ID of the profile for loading it
    time_in_arena: 300 # Time spent by the user inside the arena
    time_out_arena: 60 # Time spent by the user outside the arena
    ignore_robot_probabilities: [1, 0] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.9, 0.1] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 10 # Touch frequency of the user
    generate_touch_prob: [1, 0] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [1,0] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0.01, 0.99] # Probability of showing a ball to the robot for stimulating playing

  - id: engaged_touch # ID of the profile for loading it
    time_in_arena: 300 # Time spent by the user inside the arena
    time_out_arena: 60 # Time spent by the user outside the arena
    ignore_robot_probabilities: [0.01, 0.99] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.05, 0.95] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 10 # Touch frequency of the user
    generate_touch_prob: [1, 0] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [1,0] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0.01, 0.99] # Probability of showing a ball to the robot for stimulating playing

  - id: engaged_answers # ID of the profile for loading it
    time_in_arena: 300 # Time spent by the user inside the arena
    time_out_arena: 60 # Time spent by the user outside the arena
    ignore_robot_probabilities: [1, 0] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.9, 0.1] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 60 # Touch frequency of the user
    generate_touch_prob: [0.1, 0.9] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [0,1] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0.01, 0.99] # Probability of showing a ball to the robot for stimulating playing

################################################
############   DISENGAGED PROFILES   ###########
################################################

  - id: neutral # ID of the profile for loading it
    time_in_arena: 300 # Time spent by the user inside the arena
    time_out_arena: 300 # Time spent by the user outside the arena
    ignore_robot_probabilities: [0.9, 0.1] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.5, 0.5] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 60 # Touch frequency of the user
    generate_touch_prob: [0.5, 0.5] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [0.5, 0.5] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0.0001, 0.9999] # Probability of showing a ball to the robot for stimulating playing

################################################
############   DISENGAGED PROFILES   ###########
################################################

  - id: disengaged_complete # ID of the profile for loading it
    time_in_arena: 60 # Time spent by the user inside the arena
    time_out_arena: 300 # Time spent by the user outside the arena
    ignore_robot_probabilities: [0.05, 0.95] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.01, 0.99] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 60 # Touch frequency of the user
    generate_touch_prob: [0.1, 0.9] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [0,1] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0, 1] # Probability of showing a ball to the robot for stimulating playing

  - id: disengaged_touch # ID of the profile for loading it
    time_in_arena: 60 # Time spent by the user inside the arena
    time_out_arena: 300 # Time spent by the user outside the arena
    ignore_robot_probabilities: [0.05, 0.95] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.01, 0.99] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 10 # Touch frequency of the user
    generate_touch_prob: [0.25, 0.75] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [0.25,0.75] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0, 1] # Probability of showing a ball to the robot for stimulating playing

  - id: disengaged_answers # ID of the profile for loading it
    time_in_arena: 60 # Time spent by the user inside the arena
    time_out_arena: 300 # Time spent by the user outside the arena
    ignore_robot_probabilities: [0.5, 0.5] # Probabilities used to ignore the robot or reply it during the game [Ignore False, Ignore True]
    answers_probabilities: [0.2, 0.8] # Probability of giving a correct answer when playing [CORRECT, INCORRECT]
    touch_frequency: 60 # Touch frequency of the user
    generate_touch_prob: [0.1, 0.9] # Probability of generate a simulated touch [True, False]
    response_interaction_request: [0,1] # Probabilities used for testing if the user replies to interaction request [REPLY, NOT REPLYING]
    balls_probabilities: [0, 1] # Probability of showing a ball to the robot for stimulating playing